tests/containers/bci_prepare.pm

```diff
@@ -86,7 +86,7 @@ sub prepare_virtual_env {
             zypper_call("in saltbundlepy-base venv-salt-minion");
             assert_script_run("mkdir -p ./bci/bin");
             assert_script_run("ln -s /usr/lib/venv-salt-minion/bin/activate ./$virtualenv");
-        } elsif ($version =~ /15\.[4-7]/) {
+        } elsif ($host_distri =~ /sles/i && $version =~ /15\.[4-7]/) {
```

Comment:
I think this goes too far. On Leap 15.6 we still need the `$python` and `$pip` setting set, and the other packages installed. Perhaps only exclude the `SUSEConnect` part of this block?